languages/go/sample-basics.go

```diff
@@ -0,0 +1,84 @@
+
+package main
+
+import "fmt"
+
+func main() {
+  fmt.Println("+++ Sample Basics in Go +++")
+
+  println("Generating sample slice of ints...")
```

Comment:
Let's remove these `...` ellipsis here so there's never any confusion/oversight with the built-in varargs operator in variadic functions/calls.